Backpack/src/main/java/net/skyscanner/backpack/calendar/view/MonthView.kt

```diff
@@ -186,6 +186,7 @@ internal class MonthView @JvmOverloads constructor(
     controller?.let {
       drawMonthTitle(canvas)
       drawDaysInMonth(it, canvas)
+      contentDescription = monthHeaderString
```

Comment:
Why is this necessary?

Reply:
Unllike `textview` ,  the `paint` object does not set the content description automatically. Adding this will speak the month name correctly.